libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -942,7 +910,9 @@ void NavEKF3::UpdateFilter(void)
 */
 void NavEKF3::checkLaneSwitch(void)
 {
-    uint32_t now = AP_HAL::millis();
+    AP::dal().log_event3(AP_DAL::Event3::checkLaneSwitch);
+
+    uint32_t now = AP::dal().millis();
```

Comment:
```suggestion
    const uint32_t now = AP::dal().millis();
```

Reply:
i don't think we need to constify all of the ekf in this PR